src/System.Management.Automation/engine/CultureVariable.cs

```diff
@@ -25,7 +25,7 @@ public override object Value
             get
```

Comment:
@iSazonov, your last commit had 32 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

WSMan Config Provider.Set-Item tests.Set-Item on non-existent property should fail
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.InvalidOperationException,Microsoft.PowerShell.Commands.SetItemCommand' to be thrown, but the FullyQualifiedErrorId was 'WsManError'. from D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1:213 char:15
    +             { Set-Item $testPluginPath\foo "bar" } | Should -Throw -E ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 213
213:             { Set-Item $testPluginPath\foo "bar" } | Should -Throw -ErrorId "System.InvalidOperationException,Microsoft.PowerShell.Commands.SetItemCommand"
```
WSMan Config Provider.Set-Item tests.Set-Item on plugin InitializationParameters 'PSVersion' property with '6.0.0' should succeed
```powershell
The WinRM client received an HTTP server error status (500), but the remote service did not include any other information about the cause of the failure. 
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 254
254:             Set-Item "$testPluginPath\InitializationParameters\$property" $value -WarningAction SilentlyContinue

```
WSMan Config Provider.Set-Item tests.Set-Item on plugin InitializationParameters 'psvERSION' property with '5.1' should succeed
```powershell
The WinRM client received an HTTP server error status (500), but the remote service did not include any other information about the cause of the failure. 
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 254
254:             Set-Item "$testPluginPath\InitializationParameters\$property" $value -WarningAction SilentlyContinue

```
WSMan Config Provider.Set-Item tests.Set-Item on plugin Quotas 'IdleTimeoutms' property with '61000' should succeed
```powershell
The WinRM client received an HTTP server error status (500), but the remote service did not include any other information about the cause of the failure. 
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 265
265:             Set-Item "$testPluginPath\Quotas\$property" $value -WarningAction SilentlyContinue

```
WSMan Config Provider.Set-Item tests.Set-Item on plugin Quotas 'iDLEtIMEOUTMS' property with '7200000' should succeed
```powershell
The WinRM client received an HTTP server error status (500), but the remote service did not include any other information about the cause of the failure. 
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 265
265:             Set-Item "$testPluginPath\Quotas\$property" $value -WarningAction SilentlyContinue

```